src/jvmTest/kotlin/io/github/detekt/sarif4k/SarifSerializerTest.kt

```diff
@@ -73,22 +73,22 @@ class SarifSerializerTest {
     @Test
     fun `serialize kotlin data class to minified json`() {
         val actual = SarifSerializer.toMinifiedJson(sarifSchema210)
-        val expected = File(SarifSerializerTest::class.java.getResource("/test.sarif.json").path).readText()
+        val expected = getResource("/test.sarif.json").readText()
         assertEquals(expected.replace("\\s".toRegex(), ""), actual)
     }
 
     @Test
     fun `serialize kotlin data class to json`() {
-        val actual = SarifSerializer.toJson(sarifSchema210).lines()
-        val expected = File(SarifSerializerTest::class.java.getResource("/test.sarif.json").path).readLines()
+        val actual = SarifSerializer.toJson(sarifSchema210)
+        val expected = getResource("/test.sarif.json").readText()
         assertEquals(expected, actual)
     }
 
     @Test
     fun `deserialize json into kotlin data class`() {
-        val actual = SarifSerializer.fromJson(
-            File(SarifSerializerTest::class.java.getResource("/test.sarif.json").path).readText()
-        )
+        val actual = SarifSerializer.fromJson(getResource("/test.sarif.json").readText())
         assertEquals(sarifSchema210, actual)
     }
-}
\ No newline at end of file
+}
+
+private fun getResource(path: String) = File(SarifSerializerTest::class.java.getResource(path)!!.path)
```

Comment:
Don't assume File paths, there's better API without assumptions:
```suggestion
private fun getResource(path: String) = SarifSerializerTest::class.java.getResourceAsStream(path)!!.path

private fun InputStream.readText(): String =
    use { it.reader().readText() }
```

Reply:
Fixed